<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>417</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	border: 3px solid #46C2CB;
	border-radius: 25px;
	background-color: &quot;#453C67&quot;;
}

QPushButton{
	border-radius: 20px;
	background-color: &quot;#6D67E4&quot;;
	color: &quot;#46C2CB&quot;;
	font-weight: bold;
}
QPushButton:hover{
	background-color: &quot;#46C2CB&quot;;
	color: &quot;#6D67E4&quot;;
}

#Predicao{
	color: &quot;#46C2CB&quot;;
	font-weight:  bold;
}</string>
   </property>
   <widget class="QPushButton" name="CarregarImagemButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Carregar Imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PredizerButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Predizer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Predicao">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Predição</string>
    </property>
   </widget>
   <widget class="QLabel" name="Imagem">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FecharJanela">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
